lib/diagnostic.js

```diff
@@ -6,30 +6,60 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (paragraph) {
```

Comment:
This function only splits strings using a space.  It needs to use the regular expression ```/\s+/``` to split using all whitespace (spaces, tabs, line breaks, etc.).